File: implementation/scipamato/core/sync/src/test/java/ch/difty/scipamato/core/sync/jobs/newsletter/NewStudyItemWriterTest.java
package ch.difty.scipamato.core.sync.jobs.newsletter;

import org.jooq.DSLContext;

import ch.difty.scipamato.core.sync.jobs.AbstractItemWriterTest;

class NewStudyItemWriterTest extends AbstractItemWriterTest<PublicNewStudy, NewStudyItemWriter> {

    @Override
    protected NewStudyItemWriter newWriter(DSLContext dslContextMock) {
        return new NewStudyItemWriter(dslContextMock);
    }
}

File: implementation/scipamato/core/sync/src/test/kotlin/ch/difty/scipamato/core/sync/jobs/newsletter/NewStudyItemWriterTest.kt
package ch.difty.scipamato.core.sync.jobs.newsletter

import ch.difty.scipamato.core.sync.jobs.AbstractItemWriterTest
import org.jooq.DSLContext

internal class NewStudyItemWriterTest : AbstractItemWriterTest<PublicNewStudy, NewStudyItemWriter>() {
    override fun newWriter(dslContextMock: DSLContext?) = NewStudyItemWriter(dslContextMock)
}